osu.Game/Screens/OnlinePlay/Multiplayer/Match/MatchStartControl.cs

```diff
@@ -14,12 +14,15 @@
 using osu.Framework.Threading;
 using osu.Game.Online.Multiplayer;
 using osu.Game.Online.Multiplayer.Countdown;
+using osu.Game.Online.Rooms;
 using osuTK;
 
 namespace osu.Game.Screens.OnlinePlay.Multiplayer.Match
 {
     public class MatchStartControl : MultiplayerRoomComposite
     {
+        public readonly IBindable<PlaylistItem> SelectedItem = new Bindable<PlaylistItem>();
```

Comment:
Please add xmldoc. I'm still confused as to the difference between `Current` and `Selected` items.